app/src/main/java/com/pvp/app/di/ServiceModule.kt

```diff
@@ -32,6 +34,10 @@ interface ServiceModule {
     @Singleton
     fun bindAuthenticationService(service: AuthenticationServiceImpl): AuthenticationService
 
+    @Binds
+    @Singleton
+    fun bindExerciseService(service: ExerciseServiceImpl): ExerciseService
```

Comment:
ABCDE, it should be after `bindConfiguration`.

Reply:
Yup gonna relearn the alphabet, mb